app/domain/authentication/webservice.rb

```diff
@@ -13,7 +13,7 @@
 
 module Authentication
   class Webservice < ::Dry::Struct
-    constructor_type :schema
+    constructor_type
```

Comment:
`transform_types` have been removed in the latest Dry Struct version.  It looks like schema is the default transformation, so we should be able to remove this line completely.

Reply:
Removed line - Thanks!